src/fsharp/FSharp.Core.nuget/FSharp.Core.nuspec

```diff
@@ -5,50 +5,26 @@
         <iconUrl>http://fsharp.org/img/logo.png</iconUrl>
         <language>en-US</language>
         <dependencies>
-            <group targetFramework=".NETStandard1.6">
-                <dependency id="System.Collections" version="4.0.11" />
-                <dependency id="System.Console" version="4.0.0" />
-                <dependency id="System.Diagnostics.Debug" version="4.0.11" />
-                <dependency id="System.Diagnostics.Tools" version="4.0.1" />
-                <dependency id="System.Globalization" version="4.0.11" />
-                <dependency id="System.IO" version="4.1.0" />
-                <dependency id="System.Linq" version="4.1.0" />
-                <dependency id="System.Linq.Expressions" version="4.1.0" />
-                <dependency id="System.Linq.Queryable" version="4.0.1" />
-                <dependency id="System.Net.Requests" version="4.0.11" />
-                <dependency id="System.Reflection" version="4.1.0" />
-                <dependency id="System.Reflection.Extensions" version="4.0.1" />
-                <dependency id="System.Resources.ResourceManager" version="4.0.1" />
-                <dependency id="System.Runtime" version="4.1.0" />
-                <dependency id="System.Runtime.Extensions" version="4.1.0" />
-                <dependency id="System.Runtime.Numerics" version="4.0.1" />
-                <dependency id="System.Text.RegularExpressions" version="4.1.0" />
-                <dependency id="System.Threading" version="4.0.11" />
-                <dependency id="System.Threading.Tasks" version="4.0.11" />
-                <dependency id="System.Threading.Tasks.Parallel" version="4.0.1" />
-                <dependency id="System.Threading.Thread" version="4.0.0" />
-                <dependency id="System.Threading.ThreadPool" version="4.0.10" />
-                <dependency id="System.Threading.Timer" version="4.0.1" />
-            </group>
             <group targetFramework=".NETStandard2.0">
+                <dependency id="NETStandard.Library" version="2.0.3" />
```

Comment:
Same question as before here - is this necessary to specify, since NETStandard.Library is already deployed by the SDK?

Reply:
@cartermp I'm not sure, I think your intuition that it's not is correct.